packages/babel-cli/test/index.js

```diff
@@ -7,6 +7,7 @@ const escapeRegExp = require("lodash/escapeRegExp");
 const merge = require("lodash/merge");
 const path = require("path");
 const fs = require("fs");
+const { chmod } = require("../src/babel/util");
```

Comment:
```suggestion
const { chmod } = require("../lib/babel/util");
```
We should test transpiled files